webapi/src/main/resources/knora-ontologies/knora-base.ttl

```diff
@@ -527,6 +527,19 @@
     :subjectClassConstraint :Resource ;
     :objectClassConstraint  :Value .
 
+###  http://www.knora.org/ontology/knora-base#hasCopyrightAttribution
+
+:hasCopyrightAttribution
+    rdf:type                  owl:DatatypeProperty ;
+    rdfs:comment              "The copyright statement that gives credit to the original author."@en ;
+    :objectDatatypeConstraint xsd:string .
+
+###  http://www.knora.org/ontology/knora-base#hasLicense
+
+:hasLicense
+    rdf:type                  owl:DatatypeProperty ;
+    rdfs:comment              "Specifies the terms under which a work can be used. This statement may be a reference to a well-known license, such as Creative Commons (e.g. 'CC BY-SA') or a custom license."@en ;
```

Comment:
Sidenote: for a nicer reading experience you can add multi-line strings with `"""  """ `in ttl.